src/screens/Events/User/eventHelpers.js

```diff
@@ -18,3 +18,6 @@ export const registerForEvent = async (data) =>
 
 export const updateEvent = async (event) =>
   (await axios.put(`/api/events/${event._id}`, event)).data;
+
+export const getEventVolunteersList = async (event) => 
+  await axios.get(`/api/events/${event._id}`);
```

Comment:
Can we have the API like `axios.get(`/api/events/${event._id}`)/volunteers;` ? Doing a get on the event_id should not ideally return the list of volunteers, that's unexpected.